package.json

```diff
@@ -8,9 +8,13 @@
   ],
   "scripts": {
     "test": "wireit",
+    "dev": "wireit",
     "format": "prettier \"./packages/*/src/**/*.ts\" --write"
   },
   "dependencies": {
+    "@11ty/eleventy": "^1.0.2",
```

Comment:
I kinda figured it didn't matter, since it all goes to the same place anyway, and these are private packages. But I guess for organization it's still a bit useful. Fixed in https://github.com/webcomponents/webcomponents.org/pull/1334